check_sites.py

```diff
@@ -18,33 +18,48 @@
 import os
 from publicsuffix2 import PublicSuffixList
 
+def find_diff_sets(old_sets, new_sets):
+    diff_sets = {}
+    subtracted_sets = {}
+    difference = set(old_sets)^set(new_sets)
+    for diff in difference:
+        if diff in new_sets:
+            diff_sets[diff] = new_sets[diff]
+        else:
+            subtracted_sets[diff] = old_sets[diff]
+    return diff_sets, subtracted_sets
+
 
 def main():
     args = sys.argv[1:]
-    inputFile = 'first_party_sets.JSON'
-    inputPrefix = ''
-    opts, _ = getopt.getopt(args, "i:", ["data_directory="])
+    input_file = 'first_party_sets.JSON'
+    input_prefix = ''
+    with_diff = False
+    opts, _ = getopt.getopt(args, "i:", ["data_directory=", "with_diff="])
```

Comment:
```suggestion
    opts, _ = getopt.getopt(args, "i:", ["data_directory=", "with_diff"])
```

No need to expect an argument for this option, since we can use the option's presence/absence as the bool.